experimental/MatroidRealizationSpaces/docs/src/introduction.md

```diff
@@ -31,4 +31,55 @@ If ``B`` is the polynomial ring `ambient_ring(RS)`, ``I`` the ideal
 `inequations(RS)`, then the coordinate ring of the realization space
 ``\mathcal{R}(M)`` is isomorphic to ``U^{-1}B/I``.  
 
+### Matroid realization spaces as affine schemes
+Every `MatroidRealizationSpace` is an instance of an affine scheme.  
+For those cases where implementations exist, the
+entire functionality provided for `AbsAffineScheme`s applies 
+to matroid realization spaces. For example:
+```julia
```

Comment:
```suggestion
```jldoctest
```
If the runtime is not absurd high, it would be great to make this a doctest instead, to make sure that this does not bitrot